src/wr_logic_ai/nodes/state_machine/state_machine.py

```diff
@@ -1,95 +1,127 @@
+#!/usr/bin/env python
+
 from statemachine import StateMachine, State
-from coordinate_manager import CoordinateManager
-from wr_logic_ai.msg import TargetMsg
-from wr_logic_ai.msg import NavigationStateMsg
+from wr_logic_ai.coordinate_manager import CoordinateManager
+from wr_logic_ai.msg import NavigationState, LongRangeAction, LongRangeGoal
+from wr_logic_ai.srv import EmptySrv
```

Comment:
Use `std_srv`

Reply:
Addressed in [277b00a](https://github.com/WisconsinRobotics/WRover_Software/pull/48/commits/277b00a47c8537aba6ba2aeeb6c09d9b5ea4d3eb).